app/frontend/tests/unit/components/designer/FormViewer.spec.js

```diff
@@ -293,7 +293,7 @@ describe('FormViewer.vue', () => {
     expect(wrapper.vm.canSaveDraft).toBeTruthy();
   });
 
-  it('onMounted', () => {
+  describe('onMounted', () => {
```

Comment:
Nested tests are not allowed - but tests within describe are good (as are many levels of nested describes)